### ISSUE TITLE
Remove default classes

### ISSUE THREAD

**Issue by MadReal** [MMF-FE/vue-svgicon#31]
How can I remove the default classes vue-svgicon sets every time the component is used?
I would like to be able to only use the classes I input, or nothing.

The reason for this is that the default classes will always override my classes width and height properties, and I don't like to define them in the component because I have groups of icons with certain sizes, and other groups with others, I would just be repeating myself in every component.

I also tried adding this to my main.js
```
Vue.use(VueSVGIcon, {
  defaultWidth: '',
  defaultHeight: ''
})
```
but it didn't work, becuase it will still use the .svg-icon[data-v-7ef413f4] properties.
Solutions?
Thanks

**Comment by Allenice** [comment]
#30

**Comment by MadReal** [comment]
I'm still trying to find a way around this... How can we set sizes in media query without using an !important everywhere?

**Comment by Allenice** [comment]
@MadReal Fixed in v2.0.0
